library/SimplePie/Sanitize.php

```diff
@@ -404,7 +404,7 @@ public function sanitize($data, $type, $base = '')
 				$data = $this->registry->call('Misc', 'change_encoding', array($data, 'UTF-8', $this->output_encoding));
 			}
 		}
-		return $data;
+		return trim($data);
```

Comment:
@mblaney I won't remove trim here, as it's needed!
Obviously we could call `trim` each time `$data` is updated, but is this really worth it, compared to calling it once here at the end?